packages/shared/src/node-apis/redis.ts

```diff
@@ -12,14 +13,38 @@ const jsonString = string.transform((value) => JSON.parse(value));
 
 const chainAsset = uncheckedAssetAndChain.transform(({ asset }) => asset);
 
+const BitcoinDeposit = z.object({
```

Comment:
these can be `camelCase`
```suggestion
const bitcoinDeposit = z.object({
```

Reply:
Changed ☺️